src/quark-core/src/QuarkWallet.sol

```diff
@@ -171,58 +171,48 @@ contract QuarkWallet is IERC1271 {
      * @notice Execute a QuarkOperation via signature
      * @dev Can only be called with signatures from the wallet's signer
      * @param op A QuarkOperation struct
-     * @param v EIP-712 signature v value
-     * @param r EIP-712 signature r value
-     * @param s EIP-712 signature s value
+     * @param signature A EIP-712 signature
      * @return Return value from the executed operation
      */
-    function executeQuarkOperation(QuarkOperation calldata op, uint8 v, bytes32 r, bytes32 s)
+    function executeQuarkOperation(QuarkOperation calldata op, bytes calldata signature)
         external
         returns (bytes memory)
     {
-        return executeQuarkOperationWithSubmissionToken(op, op.nonce, v, r, s);
+        return executeQuarkOperationWithSubmissionToken(op, op.nonce, signature);
     }
 
     /**
      * @notice Executes a first play or a replay of a QuarkOperation via signature
      * @dev Can only be called with signatures from the wallet's signer
      * @param op A QuarkOperation struct
      * @param submissionToken The submission token for the replayable quark operation for QuarkNonceManager. This is initially the `op.nonce`, and for replayable operations, it is the next token in the nonce chain.
-     * @param v EIP-712 signature v value
-     * @param r EIP-712 signature r value
-     * @param s EIP-712 signature s value
+     * @param signature A EIP-712 signature
```

Comment:
A digital siganture, e.g. EIP-712?

Reply:
Good call. These no longer have to be EIP-712 sigs. Updated.